python/rmm/docs/conf.py

```diff
@@ -22,8 +23,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "rmm"
-copyright = "2020-2023, NVIDIA"
-author = "NVIDIA"
+copyright = f"2018-{datetime.datetime.today().year}, NVIDIA Corporation"
```

Comment:
Yes. I have tried to implement it everywhere when I see a hardcoded end year.

The start year is just wrong - I suspect a copy-paste issue. RMM has definitely been around since 2018.